org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/AdvancedNewProjectPage.xtend

```diff
@@ -211,22 +233,11 @@ class AdvancedNewProjectPage extends WizardPage {
 			}
 		}
 
-		val dependend = #[createUiProject, createIdeaProject, createWebProject]
-		if (!createIdeProject.selection && dependend.exists[selection]) {
-			val affectedProjects = dependend.filter[selection].join(', ', [text])
-			if (createIdeProject === source) {
-				reportIssue(ERROR, '''
-				Frontend projects like '�affectedProjects�' depends on '�createIdeProject.text�' project.
-				Please <a>deselect</a> these.''', [
-					dependend.forEach[selection = false]
-				])
-			} else {
-				reportIssue(ERROR, '''
-				Projects like '�affectedProjects�' depends on '�createIdeProject.text�' project.
-				Please <a>enable '�createIdeProject.text�'</a> project.''', [
-					createIdeProject.selection = true
-				])
-			}
+		if (autoSelectIdeProject) {
+			autoSelectIdeProject = false
+			reportIssue(INFORMATION, '''
+				'�createIdeProject.text�' project was automatically selected as option '�(source as Button).text�' requires it.
```

Comment:
Right, the status widget only stores one message at once. So if the user selects a project/build system combination that raises a message and then selects a UI project while Generic IDE was not selected so far, then the notice will overwrite the build system message. But the user has seen that message before.